docs/skopeo.1.md

```diff
@@ -123,6 +123,13 @@ Print the version number
   Default directory containing registry configuration, if **--registries.d** is not specified.
   The contents of this directory are documented in [containers-registries.d(5)](https://github.com/containers/image/blob/main/docs/containers-registries.d.5.md).
 
+## EXIT STATUS
+`skopeo` exists with status 0 if the operation was successfully. Error result
+in a non-zero exit code. If the input image cannot be found an exit status of
```

Comment:
(I wonder about being more precise here about the “image does not exist” vs. “image container does not exist” distinction we ran into with `oci*`, e.g. https://github.com/containers/image/pull/1675 . 

I’m leaning towards not discussing it here, because the text is correct as is, and most users don’t care most of the time, but don’t feel strongly about it at all.)

Reply:
I have no strong opinion on this (maybe someone else has?). I'm happy to work on this more but personally I think the (somewhat vague) description that is there now is okay.